### PR TITLE
Add new role Archivist

### LINKED ISSUE #2323: Add a separate Role "archivist"
The new role should be managed only global in the portal_role_manager and have the following permissions.
- View all dossiers in the state `dossier-state-offered` and `dossier-state-archived`
- View dispositions

### PR DESCRIPTION
**Pull request by phgross**
Add a new role `Archivist`, and let them see dispositions, offered and archived dossiers. Closes #2323

Additionally I've also add an optional configuration option to the deployment directive for the Archivist and Records Manager group.

For the local development (examplecontent) I've added to groups to our default LDAP with the following users:

`Records Manager`: Peter Archivar (peter.archivar)
`Archivist`: Peter Records Manager (peter.recordsmanager)

@deiferni

### CODE REVIEW

File: opengever/dossier/upgrades/20161125103319_let_archivist_see_offered_and_archived_dossiers/workflows/opengever_dossier_workflow/definition.xml [hunk 1]
```diff
@@ -0,0 +1,666 @@
...
+  </permission-map>
+  <permission-map name="Modify portal content" acquired="False">
+   <permission-role>Editor</permission-role>
+   <permission-role>Manager</permission-role>
+   <permission-role>Administrator</permission-role>
+  </permission-map>
+  <permission-map name="Remove GEVER content" acquired="True">
+  </permission-map>
+  <permission-map name="Review portal content" acquired="False">
+      <permission-role>Reviewer</permission-role>
```
Comment (deiferni) [inline opengever/dossier/upgrades/20161125103319_let_archivist_see_offered_and_archived_dossiers/workflows/opengever_dossier_workflow/definition.xml]: indentation is off here. strange, is this a generated file?

File: opengever/dossier/upgrades/20161125103319_let_archivist_see_offered_and_archived_dossiers/workflows/opengever_dossier_workflow/definition.xml [hunk 2]
```diff
@@ -0,0 +1,666 @@
...
+   <permission-role>Administrator</permission-role>
+  </permission-map>
+  <permission-map name="ftw.mail: Add Inbound Mail" acquired="False">
+   <permission-role>Contributor</permission-role>
+   <permission-role>Editor</permission-role>
+   <permission-role>Manager</permission-role>
+   <permission-role>Administrator</permission-role>
+  </permission-map>
+  <permission-map name="ftw.mail: Add Mail" acquired="True">
+      <permission-role>Manager</permission-role>
```
Comment (deiferni) [inline opengever/dossier/upgrades/20161125103319_let_archivist_see_offered_and_archived_dossiers/workflows/opengever_dossier_workflow/definition.xml]: same here

File: opengever/examplecontent/configure.zcml [hunk 3]
```diff
@@ -19,6 +19,8 @@
                            opengever.examplecontent:municipality_content
                            opengever.examplecontent:init"
       admin_unit_id="fd"
+      records_manager_group="record_managers"
```
Comment (deiferni) [inline opengever/examplecontent/configure.zcml]: could you also update examplecontent profiles with an example for this config option and policy templates?

File: docs/HISTORY.txt [hunk 4]
```diff
@@ -5,6 +5,10 @@ Changelog
 4.13.1 (unreleased)
 -------------------
 
+- Add new role Archivist, and let them see dispositions and offered and
```
Comment (deiferni) [inline docs/HISTORY.txt]: ~~,~~

**Comment by phgross** [comment]
@deiferni updated ...

**Pull request merged**